image_client.py

```diff
@@ -118,7 +118,7 @@ def upload_to_image_server(self, full_path, redacted, collection, original_path=
         r = requests.post(url, files=files, data=data)
         if r.status_code != 200:
             print(f"Image upload aborted: {r.status_code}:{r.text}")
-            self.monitoring_tools.add_imagepath_to_html(image_path=local_filename,
+            self.monitoring_tools.add_imagepath_to_html(image_path=original_path,
```

Comment:
changed image_path on monitoring report to be only original_path ,as to ignore anything in the .temp/ folder